src/assets/v3/library.11ty.js

```diff
@@ -60,7 +62,7 @@ class Test {
 module.exports = Test;
 
 async function getPolyfillNamesFrom(libraryVersion) {
-	const library = require(`polyfill-library-${libraryVersion}`);
+	const library = libraryVersion === latestLibraryVersion ? require(`polyfill-library`) : require(`polyfill-library-${libraryVersion}`);
```

Comment:
This threw an error as `polyfill-library-3.108.0` was never found.

Reply:
Just saw that `"polyfill-library-3.108.0": "npm:polyfill-library@3.108.0",` was added to package.json.

So it can be removed again.

Reply:
This is a good addition to keep around 👍